.github/workflows/release-changelog.yml

```diff
@@ -0,0 +1,49 @@
+name: Changelog vs Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
You simplified too much
* There is already a newer action
* You use only a shallow clone. The full history is used to generated versioncode from commit count

Please go with previous checkout

Reply:
Fixed